### PR TITLE
Fix Bug 1399277, quantum dev edition firstrun page implementation

### PR DESCRIPTION
**Pull request by schalkneethling**
## Description
This adds the new dev edition firstrun page.

## Issue / Bugzilla link
https://bugzilla.mozilla.org/show_bug.cgi?id=1399277

## Testing
Basic tests for responsiveness.
Firefox only.

### CODE REVIEW

File: bedrock/firefox/templates/firefox/developer-quantum-firstrun.html [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+        <h2 class="section-title"><span>{{ _('New Tools') }}</span></h2>
+
+        <div class="section-body">
+          <h3 id="devtools" class="body-title">{{ _('Firefox DevTools') }}</h3>
+
+          <img class="hero-image" src="{{ static('img/firefox/products/developer/quantum/hero-debugger-ani.gif') }}" alt="">
+          <p>
+          {% trans %}
+            The new Firefox DevTools are powerful, flexible, and best of all,
+            hackable. This includes a best-in-class javascript debugger, which
```
Comment (flodolo) [inline bedrock/firefox/templates/firefox/developer-quantum-firstrun.html]: Once again, JavaScript

**Comment by schalkneethling** [comment]
@flodolo Oh man :( So, than I need to be the bearer of even more bad news. The strings on what's new for the dev page is not exactly similar to this one neither. That is coming up in a separate pull request today.

No idea why this was not clearly communicated through to you.

**Comment by schalkneethling** [comment]
Thanks for the review @craigcook - updated.

File: bedrock/firefox/templates/firefox/developer-quantum-firstrun.html [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+</main>
+{% endblock %}
+
+{% block email_form %}{% endblock %}
+
+{# Bug 1381776 #}
+{% block update_notification %}{% endblock %}
+
+{% block js %}
+  {% javascript 'firefox-product-page' %}
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/developer-quantum-firstrun.html]: This doesn't seem to be the right JS bundle for this page (JS errors in the console from missing waypoints). Should the page have a secondary nav?

Comment (schalkneethling) [reply bedrock/firefox/templates/firefox/developer-quantum-firstrun.html]: Ah so, this was essentially copied from the product page. From the design, it does not require a secondary nav so, perhaps I do not need the JS at all. Will test locally and remove. Thanks for the heads up @alexgibson

Comment (craigcook) [reply bedrock/firefox/templates/firefox/developer-quantum-firstrun.html]: Yeah, no need for the JS if there's no subnav. I missed that on my first pass. Sorry!

**Comment by craigcook** [comment]
I notice this PR has a bunch of slightly-different-but-not-really files replacing some existing assets. Does it just need a rebase since those assets landed in master?

**Comment by schalkneethling** [comment]
> I notice this PR has a bunch of slightly-different-but-not-really files replacing some existing assets. Does it just need a rebase since those assets landed in master?

Nope, I ran them through imageOptim

**Comment by craigcook** [comment]
> Nope, I ran them through imageOptim

Hm, so did I... I thought. Did the sizes reduce much when you ran it, or maybe running it again just altered the binary just enough for git to treat it as different?

Also r+, if you want to squash commits. I'll merge this one first and you can rebase #5132 before I merge that one (since they share some bits).

**Pull request merged**